app/src/main/java/com/github/studydistractor/sdp/ui/FriendsScreen.kt

```diff
@@ -18,7 +20,9 @@ import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.github.studydistractor.sdp.friends.FriendsViewModel
+import com.github.studydistractor.sdp.ui.components.MessageCard
 
 @Composable
 fun  FriendsScreen(friendsViewModel: FriendsViewModel) {
```

Comment:
The screen is too complicated and contains too much code's lines. Please break it down into smaller components